IFC4x3/Properties/r/RatedFrequency_2e0YIS0SP8ffTho62YkBSL/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RatedFrequency_2e0YIS0SP8ffTho62YkBSL" Name="RatedFrequency" UniqueId="a802249c-01c6-48a6-976b-c860a2b8b715" PrimaryDataType="IfcFrequencyMeasure" />
```

Comment:
Reuse something like `NominalFrequency`?

Reply:
Semantically this one requires "optimum operating", so I keep this one.